.gitignore

```diff
@@ -1,2 +1,3 @@
 composer.lock
+/.idea/
```

Comment:
I think all specific IDE stuff (like .idea directory created by JetBrains' IDE shouldnt be here...) You should move that over to your local global .gitignore file... 

(see for example here http://stackoverflow.com/questions/6626136/best-practice-for-adding-gitignore-to-repo)